semeio/jobs/rft/trajectory.py

```diff
@@ -61,15 +101,38 @@ def inactive_info(self, zonemap=None):
             )
 
     def get_pressure(self):
+        """Returns the simulated pressure for the point, or -1 if
+        no simulated pressure is available
+
+        Returns:
+            float
+        """
         if self.is_active():
             return self.pressure
         return -1
 
-    def update_pressure_from_rft(self, rft):
+    def update_simdata_from_rft(self, rftfile):
+        """Fetch simulated data from an Eclipse simulation by looking up
+        binary RFT files. This requires the point to have the ijk-tuple
+        set upfront.
+
+        Args:
+            rftfile (EclRFTFile)
+        """
         if self.grid_ijk:
-            rftcell = rft.ijkget(self.grid_ijk)
-            if rftcell:
+            rftcell = rftfile.ijkget(self.grid_ijk)
+            if isinstance(rftcell, EclPLTCell):
```

Comment:
Can you change this back to `if rftcell` and just do `if` on `EclRFTCell` to avoid repeating?

Reply:
Fixed.